{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/{{cookiecutter.app_name}}/email_handler.py

```diff
@@ -0,0 +1,117 @@
+import furl
+from django.conf import settings
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils import timezone
+
+from {{cookiecutter.package_name}}.learning import models
```

Comment:
Should this be `{{cookiecutter.package_name}}.{{cookiecutter.app_name}}`?

Reply:
yes! thank you :)